### PR TITLE
support Kubernetes 1.26

### PR DESCRIPTION
**Pull request by llamerada-jp**
drop support for Kubernetes 1.23 and add support for Kubernetes 1.26

### CODE REVIEW

**Review by pluser — DISMISSED** [review]

File: client/client.go [hunk 1]
```diff
@@ -311,79 +319,96 @@ func (c *wrappedClient) RESTMapper() meta.RESTMapper {
...
+	subResource string
 }
 
-var _ client.StatusWriter = &wrappedStatusWriter{}
+var _ client.SubResourceClient = &wrappedSubResourceClient{}
+
+func (c *wrappedSubResourceClient) Get(ctx context.Context, obj client.Object, subResource client.Object, opts ...client.SubResourceGetOption) error {
+	// - This method is not using from TopoLVM at this time.
+	// - To implement this method, tests are required, but there are no implemented similar
+	//   tests upstream, and it consumes a lot of costs to study it.
```
Comment (peng225) [inline client/client.go:332]:
```suggestion
	// - This method is currently not used in TopoLVM.
	// - To implement this method, tests are required, but there are no similar tests
	//   implemented in the upstream code, and we will need a lot of effort to study it.
```

File: client/client.go [hunk 2]
```diff
@@ -311,79 +319,96 @@ func (c *wrappedClient) RESTMapper() meta.RESTMapper {
...
+	return fmt.Errorf("wrappedSubResourceClient.Get is not implemented")
+}
+
+func (c *wrappedSubResourceClient) Create(ctx context.Context, obj client.Object, subResource client.Object, opts ...client.SubResourceCreateOption) error {
+	// - This method is not using from TopoLVM at this time.
+	// - To implement this method, tests are required, but there are no implemented similar
+	//   tests upstream, and it consumes a lot of costs to study it.
```
Comment (peng225) [inline client/client.go:339]: ditto

File: docs/maintenance.md [hunk 3]
```diff
@@ -63,6 +63,13 @@ At last, make it tidy.
 $ go mod tidy
 ```
 
+Regenerate manifests using new controller-tools.
+
+```
```
Comment (peng225) [inline docs/maintenance.md:68]:
```suggestion
```bash
```

**Review by pluser — APPROVED** [review]

**Review by peng225 — APPROVED** [review]

**Comment by pluser** [comment]
ref. issue #623

**Pull request merged**